backend/controllers/homepage/homepage.ts

```diff
@@ -51,7 +53,7 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
       description: 'View unlock lists, all appointments and COVID units, manage attendance and add bulk appointments.',
       href: '/manage-prisoner-whereabouts',
       roles: null,
-      enabled: () => whereaboutsConfig?.enabled,
+      enabled: () => whereaboutsConfig?.enabled && !activities.enabled_prisons.split(',').includes(activeCaseLoadId),
     },
```

Comment:
Slight oddity here?  Whereabouts includes both activities and appointments sub-tiles. Switching if off based on the activities enabled prisons only is tying the rollouts together? Maybe a discussion with Grace, Jon or Joe? If activities is enabled and appointments is not - could show Whereabouts, but not the activity-tiles? And vice-versa is appointments is enabled but activities is not.